lib/req_athena.ex

```diff
@@ -109,30 +112,36 @@ defmodule ReqAthena do
 
   defp run(%Request{options: %{athena: query} = options} = request) do
     url = "https://athena.#{options.region}.amazonaws.com"
+    cache_query = options[:cache_query] || false
```

Comment:
I think it should default to true?

```suggestion
    cache_query = Keyword.get(options, :cache_query, true)
```

Reply:
I was using in the opposite way, I'll fix that